website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -146,201 +94,143 @@ Modify `roles/memcached/tasks/main.yml` to look like the following:
-```
-
-### Additional Ansible Debug
-
-Occasionally while developing additional debug in the Operator logs is nice to have.
-Using the memcached operator as an example, we can simply add the
-`"ansible.sdk.operatorframework.io/verbosity"` annotation to the Custom
-Resource with the desired verbosity.
-
-```yaml
-apiVersion: "cache.example.com/v1alpha1"
-kind: "Memcached"
-metadata:
-  name: "memcached-sample"
-  annotations:
-    "ansible.sdk.operatorframework.io/verbosity": "4"
-spec:
-  size: 4
+NAME                     DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+memcached-operator       1         1         1            1           1m
 ```
 
-### Update the size
+#### Create Memcached Resource
 
-Change the `spec.size` field in the memcached CR from 3 to 4 and apply the
-change:
+Create the resource, the operator will do the rest.
 
 ```sh
-$ cat config/samples/cache_v1alpha1_memcached.yaml
-apiVersion: "cache.example.com/v1alpha1"
-kind: "Memcached"
-metadata:
-  name: "memcached-sample"
-spec:
-  size: 4
-
 $ kubectl apply -f config/samples/cache_v1alpha1_memcached.yaml
 ```
 
-Confirm that the operator changes the deployment size:
+Verify that Memcached pods are created
 
 ```sh
-$ kubectl get deployment
-NAME                DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-memcached-sample    4         4         4            4           5m
+$ kubectl get pods
+
+NAME                                                     READY   STATUS    RESTARTS   AGE
+memcached-operator-controller-manager-7b667d9979-4jkfb   2/2     Running   0          14s
+memcached-sample-memcached-6456bdd5fc-8zgjf              1/1     Running   0          5s
+memcached-sample-memcached-6456bdd5fc-hjkrp              1/1     Running   0          5s
+memcached-sample-memcached-6456bdd5fc-mcqc5              1/1     Running   0          5s
 ```
 
-### Cleanup
+#### Cleanup
 
 Clean up the resources:
 
 ```sh
 $ make undeploy
 ```
 
-**NOTE** Additional CR/CRD's can be added to the project by running the command `operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=AppService`
-For more information, refer to the [`create api` CLI][create_api_cli] doc.
+## Next Steps
+
+We recommend reading through the our [Ansible development
+section][ansible-developer-tops] for tips and tricks, including how to
+run the operator locally.
+
+In this tutorial, the scaffolded `watches.yaml` could be used as-is, but
+has additional optional features. See [watches reference][ansible-watches].
+
+For brevity, some of the scaffolded files were left out of this guide.
+See [Scaffolding Reference][layout-doc]
+
+This example built a namespaced scope operator, but Ansible operators
+can also be used with cluster-wide scope. See the [operator scope][operator-scope] documentation.
+
+OLM will manage creation of most if not all resources required to run your operator, using a bit of setup from other operator-sdk commands. Check out the [OLM integration guide][quickstart-bundle].
```

Comment:
Could we not be more directly and just say what is the next step. Something now read X to check Z

Reply:
At this point, the user will have just read through the tutorial which is pretty sparse. If they know what they need to do next and just need a link they can use the left-bar navigation, but this section is to give them hints on what they might need to do next, not just links.